src/about/newtab/newtab/wallpapers.js

```diff
@@ -0,0 +1,107 @@
+/* @flow */
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {Effects, html, thunk, forward} from "reflex"
+import {merge, tagged} from "../../../common/prelude"
+import {Style, StyleSheet} from "../../../common/style";
+import * as Unknown from "../../../common/unknown";
+
+import hardcodedWallpaper from "../wallpaper.json";
+
+/*::
+import type {Address, DOM} from "reflex"
+import type {Model, Action, ID} from "./wallpapers"
+import * as Wallpaper from "./wallpaper";
```

Comment:
> No, not the type annotations. I mean you're not importing the Wallpaper module, but you're calling Wallpaper.view below.

Oh right, you're absolutely right! It was a good catch. Which reminded me I forgot to check `about:` things to make sure i did not broke them.

P.S.: Pushed fix for this.